extensions/cornerstone-dicom-seg/src/viewports/_getStatusComponent.tsx

```diff
@@ -34,13 +34,10 @@ export default function _getStatusComponent({ isHydrated, onStatusClick }) {
           <span className="ml-1">SEG</span>
         </div>
         {!isHydrated && (
-          <div
-            className="bg-primary-main hover:bg-primary-light ml-1 cursor-pointer rounded px-1.5 hover:text-black"
-            // Using onMouseUp here because onClick is not working when the viewport is not active and is styled with pointer-events:none
-            onMouseUp={onStatusClick}
-          >
-            {loadStr}
-          </div>
+          <ViewportActionButton
```

Comment:
Is there any reason that the names/values can't be common and the getStatusComponent be moved up a layer?  That way the same status component would be used for both seg and sr, but with differing actions on the click.  That was kind of the point of using the toolbar.  I was not suggestion changing the SEG version to be common yet, although definitely using the same component for the button is nice to do.

Reply:
Sorry, I wasn't paying enough attention - please disregard.